Gemfile

```diff
@@ -2,4 +2,7 @@
 
 source "https://rubygems.org"
 
-# gem "rails"
+gem "pg"
+gem "rspec"
+gem "simplecov"
+gem "bcrypt"
```

Comment:
add extra line at end of file to avoid this red symbol.
We leave one extra line at end of file as a convention